### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by JagaRamesh** [sheldens/intro-data-capstone-biodiversity#3]
Excellent job with the coding and the powerpoint presentation!

The matplotlib plots are neat with proper title, xlabel and ylabel defined. 

For "Endangerment of species" analysis, the question "Are certain types of species more likely to be endangered?" is not answered. I have created a separate issue for the same. 
The recommendation to conservationists to monitor Mammals & Birds is accurate.

For Foot & Mouth disease analysis, the sample size calculation and the no of weeks needed for observation is incorrect. Please check the issue created for more details.

Overall Good job!!! KEEP IT UP.

**Comment by sheldens** [comment]
Do I need to correct my errors and resubmit to receive full credit for the course?

**Comment by JagaRamesh** [comment]
Not really. The review and feedback is for your awareness .  Congratulations in completing the course!!
